src/utils/aria.ts

```diff
@@ -12,32 +12,30 @@ export const ariaLabelOrHidden = (ariaLabel?: string) =>
 export const ariaFocusableNoLabel = (ariaLabel?: string) =>
   ifAriaNoLabel(ariaLabel) ? {} : { focusable: false };
 
+type ariaPropName = 'aria-describedby' | 'aria-labelledby' | 'aria-label';
+type ariaProp = { [key in ariaPropName]: string } | {};
 /**
- * Returns object with 'aria-describedby' which can be spread to props.
+ * Returns object with 'aria-' property which can be spread to props.
  * E.g:
  * @example
  * <Component
- *   {...getAriaDescribedBy([
+ *   {...getConditionalAriaProp('aria-describedby', [
  *      describingElement ? "id-of-describing-element" : undefined,
  *      otherDescribingElement ? "id-of-other-describing-element" : undefined,
```

Comment:
Maybe we could update the example to have the so called normal case also, as it is not required to tell the "undefined" version when using, but it is possible though.

e.g 
```tsx
{...getConditionalAriaProp('aria-label', [ariaLabel])}
```

Reply:
Good point, changed the original example to have both, a static and a conditional id.